Add powershell completions (#9307)
containers/podman: docs/source/markdown/podman-completion.1.md

podman\-completion - Generate shell completion scripts

## SYNOPSIS
**podman completion** [*options*] *bash*|*zsh*|*fish*|*powershell*

## DESCRIPTION

> The completion command allows you to generate shell completion scripts. Supported shells are **bash**, **zsh**, **fish** and **powershell**.

Reviewer: I know this is pre-existing, but we try to avoid you, we, our, etc in man pages.  Suggest:
```suggestion
The completion command generates shell completion scripts for a variety of shells. Supported shells are **bash**, **zsh**, **fish** and **powershell**.
```

To make it available in all your fish sessions run:
`podman completion fish -f ~/.config/fish/completions/podman.fish`

### POWERSHELL

> To load the completion script into your current session run:

Reviewer: ```suggestion
To load the completion script into the current session run:
```

`podman.exe completion powershell | Out-String | Invoke-Expression`

> To make it available in all your powershell sessions you have to write the

Reviewer: ```suggestion
To make it available in all powershell sessions that a user has, write the
```

> completion output to a file and source that one from you powershell profile.

Reviewer: ```suggestion
completion output to a file and source that to the user's powershell profile.
```

> You can get more information about profiles with `Get-Help about_Profiles`.

Reviewer: ```suggestion
More information about profiles is available with `Get-Help about_Profiles`.
```
